### PR TITLE
Generate junit report for stork results.

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rohit@portworx.com>

**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
To generate separate xml files for stork integration tests and make them available on the jenkins host

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:
No

**Does this change need to be cherry-picked to a release branch?**:
2.2

### CODE REVIEW

**Comment by rodrigc** [comment]
This is the same as what I did here: https://github.com/portworx/torpedo/pull/236
so I am good with these changes, as long as the travis CI failures are fixed.

**Comment by Rohit-PX** [comment]
> `testMock` seems to be undefined, leading to Travis CI failures.
> Maybe you have another commit in your workspace that hasn't been pushed yet?

You are right I need to push a commit.

**Comment by Rohit-PX** [comment]
@rodrigc I have fixed and have taken out Mock tests. Based on our discussion stork test is now using `gotestsum`

**Comment by Rohit-PX** [comment]
Tested on a cluster, logs are created on `70.0.15.202` under `/testresults`

File: test/integration_test/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
-FROM ubuntu
+FROM golang:1.11.2
```
Comment (rodrigc) [inline test/integration_test/Dockerfile]: Not a big deal, but in future you can consider bumping the golang version to something higher:

https://hub.docker.com/_/golang?tab=tags

**Pull request merged**